_docs/info/migrate_entire_pool.md

```diff
@@ -0,0 +1,266 @@
+NAME                  USED  AVAIL  REFER  MOUNTPOINT
+tank2                1.58G  13.4G    24K  /tank2
+tank2/nfs             879M  13.4G   879M  /nfs
+tank2/vm-100-disk-0   734M  13.4G   734M  -
+
+local-host $ zfs list -r -t snapshot tank2
+NAME                              USED  AVAIL  REFER  MOUNTPOINT
+tank2@replication                    0      -    24K  -
+tank2/nfs@replication                0      -   879M  -
+tank2/vm-100-disk-0@replication      0      -   734M  - z
+```
+
+**Remote operation**
+
+``` terminal
+local-host $ sudo zfs send -R tank1@replication | ssh <remote-host> sudo zfs recv -Fdu tank2
+
+local-host $ ssh <remote-host> zfs list -r tank2
+NAME                  USED  AVAIL  REFER  MOUNTPOINT
+tank2                1.58G  13.4G    24K  /tank2
+tank2/nfs             879M  13.4G   879M  /nfs
+tank2/vm-100-disk-0   734M  13.4G   734M  -
+
+local-host $ ssh <remote-host> zfs list -r -t snapshot tank2
+NAME                              USED  AVAIL  REFER  MOUNTPOINT
+tank2@replication                    0      -    24K  -
+tank2/nfs@replication                0      -   879M  -
+tank2/vm-100-disk-0@replication      0      -   734M  - z
+```
+
+After we have insured our destination pool contains the same as our source pool we can safely destroy the replication snapshots on both source and destination pools as well as destroying the old source pool and recreate a new source pool. As stated at the beginning if you want to use another name for the new source pool at extra operation is needed which is shown at the end.
+
+**Local operation**
+
+``` terminal
+local-host $ sudo zfs destroy -r tank1@replication
+local-host $ sudo zfs destroy -r tank2@replication
+
+local-host $ sudo zpool destroy -f tank1
+
+local-host $ sudo zpool create tank1 c1t3d0
+```
+
+**Remote operation**
+
+``` terminal
+local-host $ sudo zfs destroy -r tank1@replication
+local-host $ ssh <remote-host> sudo zfs destroy -r tank2@replication
+
+local-host $ sudo zpool destroy -f tank1
+
+local-host $ sudo zpool create tank1 c1t3d0
+```
+
+We should now have a new pool which awaits data from our temporary pool so we basically reverse the operation used prior.
+
+**Local operation**
+
+``` terminal
+local-host $ sudo zfs snapshot -r tank2@replication
+
+local-host $ sudo zfs send -R tank2@replication | zfs recv -Fdu tank1
+
+local-host $ zfs list -r tank1
+NAME                  USED  AVAIL  REFER  MOUNTPOINT
+tank1                1.58G  13.4G    24K  /tank2
+tank1/nfs             879M  13.4G   879M  /nfs
+tank1/vm-100-disk-0   734M  13.4G   734M  -
+
+local-host $ zfs list -r -t snapshot tank1
+NAME                              USED  AVAIL  REFER  MOUNTPOINT
+tank1@replication                    0      -    24K  -
+tank1/nfs@replication                0      -   879M  -
+tank1/vm-100-disk-0@replication      0      -   734M  -
+```
+
+**Remote operation**
+``` terminal
+local-host $ ssh <remote-host> sudo zfs snapshot -r tank2@replication
```

Comment:
```suggestion
remote-host$ sudo zfs snapshot -r tank2@replication
```

how about just using `remote-host$` instead of ssh ... keeps things simpler

Reply:
Fixed in latest commit.